README.md

```diff
@@ -6,99 +6,31 @@ A Cisco Webex Teams bot is an automated user within the Webex Teams platform tha
 1. [Prerequisites](#prerequisites)
 2. [Setup Bot](#setup)
 3. [Run Bot](#run)
-4. [Understanding Bot](#understanding)
 5. [Resources](#resources)
 
 <a name="prerequisites"></a>
 ## Prerequisites
 
-### Create a Cisco Webex Teams account
+#### Create a Cisco Webex Teams account
 
 Go to the [Cisco Webex](https://www.webex.com/) website and select **Teams** in the **Sign in** drop down menu on the top right corner. Fill in your details and follow the instructions to create an account.
 
-### Install Git
+#### Install Git
 
-Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.  Find out more about git [here](https://git-scm.com/about).
+"Git is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency." [Link to install git](https://git-scm.com/download/).
 
-The git installation will depend on your machine's Operating System (OS). After installing, confirm that git is available by entering the following command on terminal.
+#### Install Python
 
-````sh
-git --version
-````
+"Python is an interpreted, high-level, general-purpose programming language" [Link to install python](https://www.python.org/downloads/).
```

Comment:
Maybe we should specify Python 3 here? Maybe 3.8 or something

Reply:
if we do not specify, it will be one less thing we need to update in future releases..

Reply:
Well, we should specify Python 3 at least, so we wouldn't need to update it for a few years haha